### PR TITLE
Add build configuration for autoversioner

### PR DESCRIPTION
**Pull request by iamsubratp**

### CODE REVIEW

File: autoversioner/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM scratch
```
Comment (hopkiw) [inline autoversioner/Dockerfile]: can you put dockerfile in container_images/autoversioner directory?

File: autoversioner/cloudbuild.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+steps:
```
Comment (hopkiw) [inline autoversioner/cloudbuild.yaml]: this config should go in the shared cloudbuild.yaml

File: autoversioner/Dockerfile (hunk 1 above)
Comment (iamsubratp) [reply autoversioner/Dockerfile]: i actually started from there. and i spent an aweful amount of time integrating gobuild with cloudbuild and container image.
the [cloud-builders](https://github.com/GoogleCloudPlatform/cloud-builders) specifications are not very clear, even that did not help. the worse thing was i could not even build the example packages that cloud-builders team have in their repo. :(
I reached out the team through [issues](https://buganizer.corp.google.com/issues/142765191); at least i was able to get some pointers around how to proceed further.
That's when i was exposed to cloudbuild parameter called "dir" that lets you run go commands from a directory of your choice. After all experimentation; i wanted a working build for this package :rofl: 

I think, now i have a better understanding of how cloudbuild organizes the code to be built and what features we have at our disposal to achieve a build.

I should be able to move the cloudbuild configuration and container definition under container_images

File: autoversioner/cloudbuild.yaml (hunk 2 above)
Comment (iamsubratp) [reply autoversioner/cloudbuild.yaml]: done

File: container_images/autoversioner/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+WORKDIR /workspace
+ADD gopath/bin /
```
Comment (hopkiw) [inline container_images/autoversioner/Dockerfile]: can we copy individually, instead, to be more explicit? I also like to copy the dockerfile into the images we produce so you can see how it was produced

Comment (iamsubratp) [reply container_images/autoversioner/Dockerfile]: done.

File: container_images/cloudbuild.yaml [hunk 4]
```diff
@@ -11,7 +11,8 @@ steps:
   args: ['build', '--tag=gcr.io/$PROJECT_ID/selinux-tools:latest', '--tag=gcr.io/$PROJECT_ID/selinux-tools:$COMMIT_SHA', './container_images/selinux-tools']
 - name: 'gcr.io/cloud-builders/docker'
   args: ['build', '--tag=gcr.io/$PROJECT_ID/build-essential:latest', '--tag=gcr.io/$PROJECT_ID/build-essential:$COMMIT_SHA', './container_images/build-essential']
-
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/autoversioner:latest', '--tag=gcr.io/$PROJECT_ID/autoversioner:$COMMIT_SHA', '--file=./container_images/autoversioner/Dockerfile', "/workspace"]
```
Comment (hopkiw) [inline container_images/cloudbuild.yaml]: i think this line needs updating to match others

**Pull request merged**